api/models/participant.py

```diff
@@ -11,7 +12,7 @@
 from .workshop import Workshop
 
 
-class Participant(Base):
+class Participant(Base, auth.models.User):
```

Comment:
I think we should use [auth.models.AbstractUser](https://github.com/django/django/blob/master/django/contrib/auth/models.py#L299) or go up to use as little as possible from the Django user functionality.